### PR TITLE
[14.0] [UPD] dotfiles: Switch to Github Actions and PIP mode

### PR DESCRIPTION
**Pull request by ivantodorovich**

### CODE REVIEW

**Comment by ivantodorovich** [comment]
@isserver1 would you be ok to change `sale_coupon_delivery_auto_refresh`'s license to AGPL-3? as the dependencies are

```
Run manifestoo -d . check-licenses
sale_coupon_delivery_auto_refresh (LGPL-3) depends on delivery_auto_refresh (AGPL-3)
sale_coupon_delivery_auto_refresh (LGPL-3) depends on sale_coupon_auto_refresh (AGPL-3)
```

**Review by chienandalu — APPROVED** [review]

File: sale_coupon_delivery_auto_refresh/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "summary": "Allows to auto-apply the coupons after adding delivery line",
     "author": "Ooops404, Cetmix, , Odoo Community Association (OCA)",
     "contributors": "Ooops404, Cetmix",
-    "license": "LGPL-3",
+    "license": "AGPL-3",
```
Comment (simahawk) [inline sale_coupon_delivery_auto_refresh/__manifest__.py:7]: what about headers?
Also, license change should imply a version bump.

Comment (ivantodorovich) [reply sale_coupon_delivery_auto_refresh/__manifest__.py:7]: There are no headers on this module, but yakes! for the version bump 😅

**Comment by pedrobaeza** [comment]
Sorry, I merged directly, as the change to GH actions can't be merged through ocabot (it gives an error).

**Pull request merged**